docs/docs/platform/quickstarts/nextjs.mdx

```diff
@@ -0,0 +1,658 @@
+
+## Prerequisites
+
+Before getting started, let's make sure that your development environment is
+ready.
+
+- You'll need **Node.js** version 10.13 or later.
+  [Install it from here](https://nodejs.org/en/).
+- You'll need the **Nhost CLI** and **Docker**. View the
+  [installation instructions](https://docs.nhost.io/platform/overview/get-started-with-nhost-cli).
+- You'll need a **Github repository**. Create a
+  [new repository](https://github.com/new).
+
+## Project setup
+
+### Create a new Nhost app
+
+First things first, we need to create a new Nhost project.
+
+So, log in to your Nhost dashboard and click the **Create your first app**
+button.
+
+![nhost-first-app](/img/quickstarts/create-app-step-1.png)
+
+Next, give your new Nhost app a name, select a geographic region for your Nhost
+services and click **Create App**.
+
+![nhost-first-app](/img/quickstarts/create-app-step-2.png)
+
+After a few seconds, you should get a PostgreSQL database, a GraphQL API with
+Hasura, file storage, and authentication set up.
+
+:::info
+You can also connect your Nhost app to a GitHub repository. When you do this, any updates you push to your code will automatically be deployed. [Learn more](https://docs.nhost.io/platform/github-integration).
+:::
+
+## Initialize the app
+
+### Create a Next.js app
+
+The simplest way to create a new Next.js application is by using the tool called
+`create-next-app`, which bootstraps a Next.js app for you without the hassle of
+configuring everything yourself.
+
+So, open your terminal, and run the following command:
+
+```bash
+npx create-next-app my-nhost-app --example "https://github.com/nhost/quickstart-nextjs"
+```
+
+:::info
+This command uses an [existing template](https://github.com/nhost/quickstart-nextjs), through the `--example` flag, which already contains the React components and pages we'll use for this guide.
+:::
+
+You can now `cd` into your project directory:
+
+```bash
+cd my-nhost-app
+```
+
+And run the development server with the following command:
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```bash
+npm run dev
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="Yarn">
+
+```bash
+yarn dev
+```
+
+  </TabItem>
+</Tabs>
+
+If everything is working fine, your Next.js development server should be running
+on port 3000. Open [http://localhost:3000](http://localhost:3000) from your
+browser to check this out.
+
+### Configure Nhost with Next.js
+
+To work with Nhost from within our Next.js app, we'll use the
+[Next.js SDK](https://github.com/nhost/nhost/tree/main/packages/nextjs) provided
+by Nhost. It's a wrapper around the
+[Nhost React SDK](https://github.com/nhost/nhost/tree/main/packages/react) which
+gives us a way to interact with our Nhost backend using React hooks.
+
+You can install the Nhost Next.js SDK with:
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```bash
+npm install @nhost/react @nhost/nextjs
```

Comment:
Don't we need `graphql` as well for our package to work? The same goes for the `yarn` command.

Reply:
Added